module/documents/actor/actor.mjs

```diff
@@ -572,6 +572,9 @@ export default class Actor5e extends SystemDocumentMixin(Actor) {
       ac.equippedShield = shields[0];
     }
 
+    // Compute cover.
+    ac.cover = this.statuses.has("coverThreeQuarters") ? 5 : this.statuses.has("coverHalf") ? 2 : 0;
```

Comment:
This will complete squash any active effects targeting cover, so we might want to do an upgrade rather than just replacement.

```suggestion
    ac.cover = Math.max(ac.cover, this.statuses.has("coverThreeQuarters") ? 5 : this.statuses.has("coverHalf") ? 2 : 0);
```